cumulusci/core/config.py

```diff
@@ -930,6 +930,25 @@ def password(self):
             password = self.scratch_info['password']
         return password
 
+    @property
+    def days(self):
+        return self.config.get('days', 7)
```

Comment:
i'd do `self.config.setdefault('days',7)` just to be extra safe, in case anybody tries to do `if days in self.config`

Reply:
I've never used setdefault but it makes sense to me.